### PR TITLE
electron-cash: Add version 4.0.15

### PR DESCRIPTION
**Pull request by erogy**

### CODE REVIEW

**Comment by erogy** [comment]
Despite having 'electron' in its name the app has nothing to do with it. 7zip fails to extract anything from the file, I tried it prior to creating this PR. While it does extract some files in temp, the running executable is still in $dir.

File: bucket/electron-cash.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/Electron-Cash/Electron-Cash/releases/download/$version/Electron-Cash-$version-portable.exe",
+                "hash": {
+                    "url": "$url.sha256sum"
+                }
+            }
+        }
```
Comment (Ash258) [inline bucket/electron-cash.json:33]:
```suggestion
        "url": "https://github.com/Electron-Cash/Electron-Cash/releases/download/$version/Electron-Cash-$version-portable.exe#/Electron-Cash.exe",
        "hash": {
            "url": "$url.sha256sum"
        }
```

File: bucket/electron-cash.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
+    "version": "4.0.15",
+    "description": "Electron Cash. Bitcoin Cash thin client.",
```
Comment (Ash258) [inline bucket/electron-cash.json:3]:
```suggestion
    "description": "A Bitcoin Cash SPV Wallet",
```

File: bucket/electron-cash.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/Electron-Cash/Electron-Cash/releases/download/4.0.15/Electron-Cash-4.0.15-portable.exe",
+            "hash": "39a472e14bfcc4accc26e16dcaad3a0bdde964a83193442e799434d94b034a13"
+        }
+    },
+    "installer": {
+        "script": "Rename-Item \"$dir\\Electron-Cash-$version-portable.exe\" \"$dir\\Electron-Cash.exe\""
+    },
```
Comment (Ash258) [inline bucket/electron-cash.json:14]:
```suggestion
    "url": "https://github.com/Electron-Cash/Electron-Cash/releases/download/4.0.15/Electron-Cash-4.0.15-portable.exe#/Electron-Cash.exe",
    "hash": "39a472e14bfcc4accc26e16dcaad3a0bdde964a83193442e799434d94b034a13",
```

**Pull request merged**